### PR TITLE
feature: Close connection after each job

### PR DESCRIPTION
**Pull request by taka-oyama**

### CODE REVIEW

File: src/SpannerServiceProvider.php [hunk 1]
```diff
@@ -96,4 +103,17 @@ protected function createAuthCache(): CacheItemPoolInterface
         $cachePath = storage_path(implode(DIRECTORY_SEPARATOR, ['framework', 'spanner']));
         return new FilesystemAdapter('auth', 0, $cachePath);
     }
+
+    protected function closeSessionAfterEachQueueJob(): void
+    {
+        $this->app->resolving('queue', function (QueueManager $queue) {
+            $queue->after(static function () {
```
Comment (zeriyoshi) [inline src/SpannerServiceProvider.php:110]:
```suggestion
        $this->app->resolving('queue', function (QueueManager $queue): void {
            $queue->after(static function (): void {
```

**Review by zeriyoshi — CHANGES_REQUESTED** [review]
Please check suggestion

File: tests/Queue/QueueEventTest.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+use Google\Cloud\Spanner\Transaction;
+use Illuminate\Bus\Dispatcher;
+use Illuminate\Database\Events\QueryExecuted;
+use Illuminate\Database\Events\TransactionBeginning;
+use Illuminate\Database\Events\TransactionCommitted;
+use Illuminate\Queue\QueueManager;
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Event;
+use Symfony\Component\Cache\Adapter\ArrayAdapter;
```
Comment (halnique) [inline tests/Queue/QueueEventTest.php:44]: Please optimize imports.

Comment (taka-oyama) [reply tests/Queue/QueueEventTest.php]: optmized.

**Review by zeriyoshi — APPROVED** [review]

**Comment by zeriyoshi** [comment]
LGTM

**Review by halnique — APPROVED** [review]

**Pull request merged**